<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mymplayer</class>
 <widget class="QMainWindow" name="Mymplayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>29</y>
      <width>631</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page_basic">
     <widget class="QLabel" name="label_name">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>230</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>user</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_showtime">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>仿宋</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>day time</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_music"/>
    <widget class="QWidget" name="page_personal"/>
    <widget class="QWidget" name="page_local">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>580</width>
        <height>341</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>歌曲名</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_local_text">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>本地音乐</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_add_music">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>70</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>更换目录</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>520</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="hs_nowMusicTime">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>520</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_play_music">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>510</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_now_time">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>540</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_pause_music">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>510</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_mymusic">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>我的音乐</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_home">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>回到主页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_localmusic">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>本地音乐</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_myfun">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>我最喜欢</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>133</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bt_jumpToWord">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>music</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_infor_music">
       <property name="text">
        <string>infomation</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="bt_start">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>510</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_pre_music">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>上一曲</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_next_music">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>510</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>下一曲</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sum_time">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>540</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>用户</string>
    </property>
    <addaction name="actionmodifi"/>
    <addaction name="actionpassward"/>
    <addaction name="actionlogout"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen_file"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>操作</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionlogout">
   <property name="text">
    <string>注销账户</string>
   </property>
  </action>
  <action name="actionopen_file">
   <property name="text">
    <string>打开歌曲目录</string>
   </property>
  </action>
  <action name="actionpassward">
   <property name="text">
    <string>修改密码</string>
   </property>
  </action>
  <action name="actionmodifi">
   <property name="text">
    <string>个人信息</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
